src/healthcheckService.js

```diff
@@ -15,11 +15,17 @@ class HealthcheckService {
   }
 
   async healthcheckHandler (req, res, next) {
-    // if (!this.pinning.ipfs.isOnline()) return res.status(503).send()
-    // const cpuFree = await new Promise((resolve, reject) => os.cpuFree(resolve))
-    // const memFree = os.freememPercentage()
-    // if (cpuFree < 0.05 || memFree < 0.20) return res.status(503).send()
-    // if (memFree < 0.20) return res.status(503).send()
+    if (!this.pinning.ipfs.isOnline()) {
+      return res.status(503).send()
+    }
+    const cpuFree = await new Promise((resolve, reject) => os.cpuFree(resolve))
+    const memFree = os.freememPercentage()
+    if (cpuFree < 0.05 || memFree < 0.20) {
+      return res.status(503).send()
+    }
+    if (memFree < 0.20) {
```

Comment:
already checked above?

Reply:
oh yeah, I commented out the previous code. I'll make a change, thanks!